docs/DeveloperGuide.md

```diff
@@ -548,8 +548,8 @@ Step 3. CONNECTIONS will not display the first person's invalid `Tag` and will n
 
 **MSS**
 
-1.  User requests to list persons
-2.  CONNECTIONS show a list of persons
+1.  User requests to list people
```

Comment:
Would `contacts` be better?

Reply:
Not sure if changing to people is better tho since 'people is used when referring to a collective group or indeterminate number, and persons serves better when referring to individuals (or a number of individuals)'

Reply:
persons isn't gramitically correct though thats why I changed it